hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -274,6 +295,22 @@ std::vector<hardware_interface::CommandInterface> GenericSystem::export_command_
     }
   }
 
+  // GPIO command interfaces
+  for (auto i = 0u; i < info_.gpios.size(); i++)
```

Comment:
We could use lamdba-function to shorten this code.

Reply:
yes, we will need some more optimizations.